Utilities/gdcmcharls/jpegmarkersegment.cpp

```diff
@@ -58,7 +58,7 @@ unique_ptr<JpegMarkerSegment> JpegMarkerSegment::CreateJpegFileInterchangeFormat
     content.push_back(static_cast<uint8_t>(params.Ythumbnail));
     if (params.Xthumbnail > 0)
     {
-        if (params.thumbnail)
```

Comment:
This is 3rd party code, so needs separate handling.

Is this reversal of logic fixed upstream already perhaps?